lib/libc/Kconfig

```diff
@@ -6,19 +6,41 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+
```

Comment:
I don't know if we have style guidelines for this, but from the diffs it doesn't appear that double line separation is used consistently here.  I'd personally rather see single blank separation consistently, as it is in the current master version of this file.